get-otp.sh

```diff
@@ -31,7 +31,9 @@ if [ ! -f configure ]; then
 fi
 
 if [ -d /usr/local/openssl ]; then
-    extra_config="--with-ssl=/usr/local/openssl/"
+    # for envs where openssl is installed from source
+    # we static-link with libcrypto
+    extra_config="--with-ssl=/usr/local/openssl --disable-dynamic-ssl-lib"
```

Comment:
Q: should this use `OPENSSL_ROOT_DIR` for the cases that do not build OpenSSL from source, like `el8`?

Reply:
for other systems. we try not to static-link openssl.